### DISCUSSION TITLE
can not run example“Cylinder Half-Submerged in Flow - UW WASIRF Twin - FOAMySees (OpenFOAM + OpenSees)”

### DISCUSSION THREAD

**Discussion by xiongmengliuliansu** [NHERI-SimCenter/SimCenterCommon#241] [Hydro-UQ]
I'm trying to run the particles coupled with opensees and open foam in the documentation but it fails, I replaced the selection file for hdr-003 but it still doesn't work, is there a specific tutorial for this?
![image](https://github.com/user-attachments/assets/14a9a492-edf7-4c54-baba-88ad7f50d57e)

![2024-10-10_222036](https://github.com/user-attachments/assets/461890dd-c9e3-454a-9ca1-e18750b0207e)

**Comment by yisangriB** [comment]
Thank you for reporting the bugs! I wanted to let you know that the main developer of Hydro-UQ is currently out of the office and will return next week. We will get back to you as soon as we can. Thanks again.

**Comment by xiongmengliuliansu** [comment]
Another question is that I installed the software by referring to the tutorials on the website but some of the modules don't work, is there something wrong with my installation?
![image](https://github.com/user-attachments/assets/f6afe809-ea07-4786-949d-8c75ca304ae0)
